app/templates/components/welcome-header.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="content">
-  <h1 class="weight-300 lead">{{t 'The open source event solution for virtual and in-person events.'}}</h1>
+  <h1 class="ui-header">{{t 'The open source event solution for virtual and in-person events.'}}</h1>
```

Comment:
I think it must not to be BOLD.

Reply:
Are you sure? if sure, then I will make changes